sdk/core/Azure.Core/src/Pipeline/Internal/RetryPolicy.cs

```diff
@@ -80,7 +80,7 @@ private async ValueTask ProcessAsync(HttpMessage message, ReadOnlyMemory<HttpPip
 
                 if (lastException != null)
                 {
-                    if (shouldRetry && message.ResponseClassifier.IsRetriableException(lastException))
```

Comment:
I guess this would be a behavior change for any libraries currently overriding original overload. I don't think there are any GA libraries that do this. Are we concerned about custom libraries?

Reply:
I don't think it is - Pavel wrote it such that the new overload immediately calls into the old version.

Reply:
Yes, my mistake.